.github/workflows/tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Test Splitter"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [main]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.11]
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3

```

Comment:
there's already a `4.x` series GH action available for `actions/checkout`: https://github.com/actions/checkout/releases/tag/v4.1.7

Reply:
bumped!